src/lib/consensus/global_slot.ml

```diff
@@ -12,7 +12,7 @@ module Make_sig (A : Wire_types.Types.S) = struct
 end
 
 module Make_str (A : Wire_types.Concrete) = struct
-  module T = Mina_numbers.Global_slot
+  module T = Mina_numbers.Global_slot_since_genesis
```

Comment:
This shouid be `Global_slot_since_hard_fork.t`. let's default consensus global_slots to since_fork unless it specified otherwise

Reply:
This one in particular needs to be since_hardfork. Otherwise any change in consensus constants will mess up the conversions (to and from slots) that are happening here

Reply:
Done.